src/classes/routes/RouterFactory.ts

```diff
@@ -0,0 +1,60 @@
+import { Router, Request, Response, NextFunction } from 'express';
+import PostImageController from './PostImageController';
+import Logger from '../Logger';
+import ConfigurationManager from '../ConfigurationManager';
+import * as multer from 'multer';
+import * as cors from 'cors';
+import { ExpressMiddleware } from '../../types';
+
+class RouterFactory {
+
+    constructor (private configurationManager: ConfigurationManager, private logger: Logger) { }
+
+    public buildImageRouter (): Router {
+        const router: Router = Router();
+        const fileInputName = this.configurationManager.getFileInputName();
+        const uploadFilePath = this.configurationManager.getUploadPath();
```

Comment:
As per the Clean Code book:
> Variables should be declared as close to their usage as possible.

Rather than declaring all at the top here, declare them right before they're used.
In this case, `fileInputName` & `uploadFilePath` should be declared before `router.post('/image...`.
OR you can move `router.post('/image...` up, right under this declaration.